Code/Editor/Controls/FolderTreeCtrl.cpp

```diff
@@ -278,17 +278,16 @@ void CFolderTreeCtrl::LoadTreeRec(const QString& currentFolder)
 
 void CFolderTreeCtrl::AddItem(const QString& path)
 {
-    QString folder;
-    QString fileNameWithoutExtension;
-    QString ext;
-
-    Path::Split(path, folder, fileNameWithoutExtension, ext);
+    AZ::IO::FixedMaxPath folder{ AZ::IO::PathView(path.toUtf8().constData()) };
+    AZ::IO::FixedMaxPath fileNameWithoutExtension = folder.Extension();
```

Comment:
Double checking:
`fileNameWithoutExtension` is set to folder.**Extension()** ?

Reply:
Yes, that should be `folder.Stem()`